### PR TITLE
RHODS: Prepare for external clusters

### PR DESCRIPTION
**Pull request by fcami**
List of changes to prepare running on non-CI clusters

```
test-path: ods jh-at-scale install_rhods
test-path: ods jh-at-scale install_ldap
test-path: ods jh-at-scale prepare_driver_cluster
test-path: ods jh-at-scale run_jupyterlab_test
test-path: ods jh-at-scale generate_plots
```

### CODE REVIEW

**Comment by kpouget** [comment]
hum, we cannot test this (with the `test-path` directives), as `/test test-pr` would use a TestPlaform cluster (=not our AWS account) and we don't want to do it anymore :/

File: testing/ods/common.sh [hunk 1]
```diff
@@ -117,6 +124,11 @@ ocm_cluster_is_ready() {
 get_osd_cluster_name() {
     cluster_role=$1
 
+    if [[ "${OSD_CLUSTER_NAME}" ]]; then
+	echo "$OSD_CLUSTER_NAME"
+	return
```
Comment (kpouget) [inline testing/ods/common.sh:129]: indentation is wrong

File: testing/ods/jh-at-scale.sh [hunk 2]
```diff
@@ -310,6 +326,36 @@ case ${action} in
         prepare
         exit 0
         ;;
+    "install_rhods")
```
Comment (kpouget) [inline testing/ods/jh-at-scale.sh:329]: indentation is wrong in the blocks below

**Review by kpouget — COMMENTED** [review]
thanks @fcami, looks good,
just here and there the indentation is wrong

File: testing/ods/common.sh (hunk 1 above)
Comment (fcami) [reply testing/ods/common.sh]: Fixed I guess

**Comment by kpouget** [comment]
one point for the CI, 0 for us 🙃 

this function should be renamed into `prepare_ocp_sutest_install_rhods`
```
prepare_ocp_sutest_cluster() {
    switch_sutest_cluster

    echo "Deploying RHODS $ODS_QE_CATALOG_IMAGE_TAG (from $ODS_QE_CATALOG_IMAGE)"

    process_ctrl::run_in_bg \
        process_ctrl::retry 5 3m \
            ./run_toolbox.py rhods deploy_ods \
                "$ODS_QE_CATALOG_IMAGE" "$ODS_QE_CATALOG_IMAGE_TAG"
}
```

and `prepare_osd_sutest_cluster` into `prepare_osd_sutest_install_rhods`

/lgtm cancel

**Comment by kpouget** [comment]
[Test passed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/469/pull-ci-openshift-psap-ci-artifacts-master-ods-jh-on-single/1553009445099802624/artifacts/jh-on-single/test/artifacts/), I think we can merge this PR. We'll address the follow up recommendations in another PR.

**Pull request merged**